runtime/compiler/optimizer/InlinerTempForJ9.cpp

```diff
@@ -5103,11 +5103,11 @@ void TR_J9InlinerUtil::checkForConstClass(TR_CallTarget *target, TR_InlinerTrace
    TR_PrexArgInfo *ecsArgInfo = target->_ecsPrexArgInfo;
    if (!ecsArgInfo) return;
 
-   TR::Compilation * comp = tracer->comp();
+   TR::Compilation * comp = TR::comp();
    bool tracePrex = comp->trace(OMR::inlining) || comp->trace(OMR::invariantArgumentPreexistence);
 
    if (tracePrex)
-      traceMsg(comp, "checkForConstClass parm for [%p] %s %s\n", callNode, callNode->getOpCode().getName(), callNode->getSymbol()->castToMethodSymbol()->getMethod()->signature(tracer->trMemory(), stackAlloc));
+      traceMsg(comp, "checkForConstClass parm for [%p] %s %s\n", callNode, callNode->getOpCode().getName(), callNode->getSymbol()->castToMethodSymbol()->getMethod()->signature(TR::comp()->trMemory(), stackAlloc));
```

Comment:
It's better to use the local variable `comp` when it's already available. `TR::comp()` is in special thread-local memory and calling it each time can add overhead.